Timefy/View/CurrentTaskComponent/CurrentTaskComponent.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  CurrentTaskComponent.swift
+//  Timefy
+//
+//  Created by Doğancan Mavideniz on 12.09.2022.
+//
+
+import UIKit
+
+class CurrentTaskComponent: UIView {
+
+    static let identifier = "CurrentTaskComponent"
+
+    @IBOutlet private weak var timeTextLabel: UILabel!
+    @IBOutlet private weak var projectNameLabel: UILabel!
+    @IBOutlet private weak var goDetailButton: UIButton!
+    @IBOutlet private weak var workTypeIconImage: UIImageView!
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        initSubviews()
+    }
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        initSubviews()
+    }
+
+    func initSubviews() {
+
+        let nib = UINib(nibName: CurrentTaskComponent.identifier, bundle: nil)
+
+        guard let view = nib.instantiate(withOwner: self, options: nil).first as?
+        UIView else { fatalError("Unable to convert nib") }
```

Comment:
bu tarz bi guard let'i ilk kez gördüm, genelde ben ikinci method gelirse koyuyorum ama burada sanki swiftlint satır sayısı linter'ına takılmış gibisin. Okumayı bence zorlaştırır :D Ben alttaki gibi yapardım :D 


```suggestion
        guard let view = nib.instantiate(withOwner: self, options: nil).first as? UIView else { fatalError("Unable to convert nib") }
```